openpose/Assets/Scripts/OSCRunwayBridge.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;

```

Comment:
Can you add a license? and you as a contributor? See: https://github.com/runwayml/examples_openFrameworks/blob/master/openpose/src/ofApp.h